e2e/testvalues/values.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	StartingTokenAmount int64  = 10_000_000
+	StartingTokenAmount int64  = 100_000_000
```

Comment:
the minimum deposit for governance is 10_000_000 so this got increased from my work on #2014. I can revert if desirable since it doesn't fit the version bump, but I figure might as well leave in